app/controllers/users_controller.rb

```diff
@@ -6,5 +6,6 @@ def index
   def show
     @individual_post = User.find(params[:id])
     @posts = Post.where(author_id: params[:id])
+    @post = @individual_post.posts.new
```

Comment:
- [Optional] Dear @0sugo, It's great that you have added the profile image for the user, But in case there is no image it would be great to have a proper placeholder text with a user name and description. It's among the primary rule of thumb when it comes to responsive design.